caffe2/CMakeLists.txt

```diff
@@ -1310,6 +1330,14 @@ if(BUILD_MOBILE_TEST)
   endforeach()
 endif()
 
+add_executable(lite_interpreter_demo ${TORCH_ROOT}/test/cpp/demos/main.cpp)
```

Comment:
Not sure if it's the convention to add an executable in CMake. I saw some benchmark or test examples. The purpose of this executable is to load a mobile module file with lite-interpreter lib, and make sure it loads and runs. @malfet do you have any suggestion?

Reply:
Wondering if we can move this to a separate CMake file? I assume this binary will be created everytime when users build PyTorch, which is not needed for non-mobile developers.

Reply:
Yeah, I don't think we should be building this one unless `BUILD_LITE_INTERPRETER` is set to true.

Reply:
+1 -- maybe also generate this only if `BUILD_LITE_INTERPRETER ` is true?

Reply:
A unit test is added in the latest commit and can be used as a light demo app. lite_interpreter_demo is not needed and removed. Thanks everyone for providing the suggestion.